main.tf

```
@@ -73,6 +87,15 @@ resource "google_dns_record_set" "redis_subdomain" {
   ttl          = var.dns_ttl
 }
 
+resource "google_dns_record_set" "redis_read_replica_subdomain" {
+  count        = local.create_private_dns ? (var.use_redis_replicas ? 1 : 0) : 0
+  managed_zone = var.dns_zone_name
+  name         = format("%s-rr.%s", var.dns_subdomain, data.google_dns_managed_zone.dns_zone.dns_name)
```

Verbosity in names are helpful when we see them isolation.

```suggestion
  name         = format("%s-replica.%s", var.dns_subdomain, data.google_dns_managed_zone.dns_zone.dns_name)
```